.travis.yml

```diff
@@ -3,12 +3,18 @@ jdk:
   - oraclejdk8
   - openjdk7
 
```

Comment:
I'm not sure why this is part of this PR. If we're going to do this change, then preferably it should be a different PR so we don't create unnecessary dependencies between PRs.